catalog.js

```diff
@@ -0,0 +1,47 @@
+var $catalog = document.getElementsByClassName('catalog');
```

Comment:
Есть еще вот такой удобный метод :) Рекомендую изучить пристально)
```suggestion
var $catalog = document.querySelector('.catalog');
```